code/modules/martial_arts/combos/cqc/kick.dm

```diff
@@ -1,7 +1,7 @@
 /datum/martial_combo/cqc/kick
 	name = "CQC Kick"
 	steps = list(MARTIAL_COMBO_STEP_HARM, MARTIAL_COMBO_STEP_HARM)
-	explaination_text = "Knocks opponent away and slows them. Will instead deal massive stamina damage, and mute opponents who are on the ground."
+	explaination_text = "Knocks opponent away and slows them. Will instead deal massive stamina damage while inflicting minor brain damage and muting opponents who are on the ground."
```

Comment:
IMO, this sounds better
```suggestion
	explaination_text = "Knocks opponent away and slows them. Will instead deal massive stamina damage, inflict minor brain damage, and mute opponents who are on the ground."
```

Reply:
Hmm. Both sound fine to me, I really have no preference either way myself. Totally fine with changing it to that 🤷‍♂️